src/main.rs

```diff
@@ -295,7 +295,7 @@ Heuristic filter selection strategies:
     8  =>  BigEnt    Highest Shannon entropy of bigrams
     9  =>  Brute     Smallest compressed size (slow)",
         )
-        .get_matches_from(wild::args());
+        .get_matches();
```

Comment:
I'll just note that there's a subtle difference to be aware of here: `wild::args()` corresponds to `env::args()`, while `get_matches()` uses `env::args_os()` (corresponding to `wild::args_os()`). It affects what happens with paths that contain invalid UTF-8.
Not suggesting to change anything, just pointing this out.